### PR TITLE
Redirect Manager:  preserve query string in external redirects

### PR DESCRIPTION
**Pull request by YegorKozlov**
Fix https://github.com/Adobe-Consulting-Services/acs-aem-commons/issues/2735

### CODE REVIEW

File: bundle/src/main/java/com/adobe/acs/commons/redirects/filter/RedirectFilter.java [hunk 1]
```diff
@@ -419,16 +419,16 @@ String evaluate(RedirectMatch match, SlingHttpServletRequest slingRequest){
...
-                String queryString = slingRequest.getQueryString();
-                if (queryString != null) {
-                    location = preserveQueryString(location, queryString);
-                }
-            }
             if(urlAdjuster != null){
                 location = urlAdjuster.adjust(slingRequest, location);
             }
         }
+        if(preserveQueryString) {
```
Comment (kwin) [inline bundle/src/main/java/com/adobe/acs/commons/redirects/filter/RedirectFilter.java:426]: nit: missing space

**Review by kwin — APPROVED** [review]
LGTM, but lacks a changelog entry

File: bundle/src/main/java/com/adobe/acs/commons/redirects/filter/RedirectFilter.java (hunk 1 above)
Comment (davidjgonzalez) [reply bundle/src/main/java/com/adobe/acs/commons/redirects/filter/RedirectFilter.java:426]:
```suggestion
        if (preserveQueryString) {

```

Comment (davidjgonzalez) [inline bundle/src/main/java/com/adobe/acs/commons/redirects/filter/RedirectFilter.java]:
```suggestion
        if (preserveQueryString) {

```

**Review by davidjgonzalez — APPROVED** [review]
I will add changelog

**Pull request merged**